extensions/gcp/tests/GCPCredentialsControllerServiceTests.cpp

```diff
@@ -89,7 +105,7 @@ TEST_F(GCPCredentialsTests, DefaultGCPCredentialsWithEnv) {
   auto temp_directory = test_controller_.createTempDirectory();
   auto path = create_mock_json_file(temp_directory);
   ASSERT_TRUE(path.has_value());
-  google::cloud::internal::SetEnv("GOOGLE_APPLICATION_CREDENTIALS", path->string());
+  setGoogleEnvironmentCredentials(path->string().c_str());
```

Comment:
```suggestion
  org::apache::nifi::minifi::utils::Environment::setEnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS", path->c_str());
```

Reply:
Now that you mention it, I remember something like this, maybe thats why I was using the internal function. Thanks for explaining :)

Reply:
Could we adapt `setEnvironmentVariable` to apply to both kinds of environment variables on Windows instead?

Reply:
I suppose that should be okay, added this change to 11f5ee48faf1d47b07c959be5d2392f35961d1fa